skbio/stats/distance/_base.py

```diff
@@ -1020,6 +1024,9 @@ class DistanceMatrix(DissimilarityMatrix):
     # Override here, used in superclass __str__
     _matrix_element_name = "distance"
 
+    read = Read()
```

Comment:
Also here. Since `DissimilarityMatrix` is a parent class of `DistanceMatrix`, I wonder if these two lines only need to exist in the former. See also my other comment.